web/frontend/src/lib/components/collapse.svelte

```diff
@@ -36,7 +36,7 @@ onMount(() => {
 });
 </script>
 
-<div class="relative w-full">
+<div class="relative w-full" id={title}>
```

Comment:
Any cases where a `title` may be a non-ideal `id` value, like spaces or anything?

Reply:
"Operations & sessions" is the only one. I can `encodeURIComponent`